vlinder/src/app/private/page.tsx

```diff
@@ -6,6 +6,7 @@ export default async function PrivatePage() {
   const supabase = createClient()
 
   const { data, error } = await supabase.auth.getUser()
+  console.log(data)
```

Comment:
only do consol log in development env variables should be used if (env == develpment) consol.log(data)

but i geuss you can just remove it here  (its for the future)